packages/storybook/stories/QuickStarts/QuickstartComposite.stories.mdx

```diff
@@ -50,6 +50,17 @@ with your new app.
 If you are on react 18 or higher and experiencing issues with starting your app,
 visit our [setup page](./?path=/docs/setup-communication-react--page) to troubleshoot the issue.
 
+If you are creating a new application run the following command to make a new app setup with the core dependencies needed for the UI library.
+You will still need to install the `uuid` package like below.
```

Comment:
I would move this out, and at the time of the instructions to create adapters have users install uuid

Reply:
Actually uuid is a dependency of our lib, so it should work without having to have users explicitly install it